variants/frontend-base/js-lint/template.rb

```diff
@@ -2,7 +2,8 @@
 # ######################################
 
 run "rm .browserslistrc"
-run "yarn add --dev eslint eslint-config-ackama eslint-plugin-node eslint-plugin-import eslint-plugin-prettier prettier prettier-config-ackama eslint-plugin-eslint-comments eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-jsx-a11y"
+run "yarn add --dev eslint eslint-config-ackama eslint-plugin-node eslint-plugin-import eslint-plugin-prettier prettier prettier-config-ackama eslint-plugin-eslint-comments eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-jsx-a11y jest eslint-plugin-testing-library eslint-plugin-jest-dom eslint-plugin-jest eslint-plugin-jest-formatting"
```

Comment:
I just realised that the `js-lint` template is applied regardless of if we're using `react` or not, which we shouldn't do.

I think the best thing to do would be to have an existence check for the `jest.config`, and then if that's there this template should do a bit of extra work.

Reply:
are you suggesting not runing the `js-lint` template at all if we're not using react - or just wrapping this new stuff in a conditional?

Reply:
ofph that's a good pont 🤦 

Ideally we probably should only be configuring react in the linting config if the react variant is used, but since we're not already, I think you can ignore this comment :D

(that's probably something to implement as a follow-up)